examples/greeter-extension/README.md

```diff
@@ -190,7 +190,7 @@ export class ChineseGreeter implements Greeter {
     /**
      * Inject the configuration for ChineseGreeter
      */
-    @inject('greeters.ChineseGreeter.options', {optional: true})
+    @config()
```

Comment:
You are changing the documentation in `README.md`, but I don't see any related changes in the TypeScript code. Is it intentional?

Reply:
The code already uses `@config`. I'm just fixing README to reflect that.